README.md

```diff
@@ -521,6 +531,16 @@ tx_hash = client.eth.solo.deposit(
 receipt = client.eth.get_receipt(tx_hash)
 
 
+# withdraw 10 ETH without using the ETH payable proxy address
+# does not require set_allowance
```

Comment:
superfluous comment. none of the withdraw functions require setting of allowance